introduction.md

```diff
@@ -5,3 +5,10 @@ I am Chris, the C-64 Hacker.  Envy my leet skillz.
 My favorite CodeSchool path is Git, and my favorite course is MasteringGitHub (of course).

```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?